certs-manager/src/main/java/io/strimzi/certs/Subject.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.certs;
+
+/**
+ * Represents the subject for a certificate
+ */
+public class Subject {
+
+    private String organizationName;
+    private String commonName;
+
+    public String organizationName() {
```

Comment:
If setters, why not getters?

But also does it even need to be mutable, or could you use a 2-param ctor?

Finally, I assume there's some prohibition on what's legal syntax in these names, is there a reason not to enforce that?

Reply:
I just added a couple of fields + subject alt names as started so I don't know how many other fields we'll need for the subject for this reason I avoided a constructor which could be larger and larger overtime.